dependencies/pom.xml

```diff
@@ -21,8 +21,31 @@
     <jetty.version>9.2.14.v20151106</jetty.version>
     <javancss.version>2.1</javancss.version>
 
```

Comment:
Here is my main concern - we use mappers as a "root" of our projects. From my point of view it should be "just another" component - mappers framerwork. We can think about moving "dependencies" (from here and ot) up to "datapad-super" level and keep all the setting independently. But anyway - that is out of scope of this issue.